api-spec/schemas/apitypes.yaml

```diff
@@ -118,6 +118,9 @@ WakuMessage:
       format: int64
     ephemeral:
       type: boolean
+    meta:
```

Comment:
How about rateLimitProof? According to https://rfc.vac.dev/spec/17/#wakumessage it can also be part of the message.
Also I saw that RPC protocol will return it, but REST does not

Reply:
To be honest, I'd like to return all the fields that are part of the wakumessage protobuffer, instead of just an specific subset

Reply:
I wonder if it's okay to do that? 
cc: @NagyZoltanPeter  what do you think if we return all fields available in the protobuffer?

Reply:
This won't be the quickest reply ever. Yes I do think we should mimic protobuf structures over REST API.